wordpress-mobile/WordPress-iOS
# Dark mode: Activity, support activity log, plans

Refs #12320. This PR adds fixes for a number of areas:

**My Site > Activity**

Cells and headers now support dark mode

![Simulator Screen Shot - iPhone Xs - 2019-08-30 at 16 29 09](https://user-images.githubusercontent.com/4780/64034439-920d3880-cb46-11e9-8980-66fff3cc56e7.png)

**Help and Support > Activity Logs**

These are now readable.

![Simulator Screen Shot - iPhone Xs - 2019-08-30 at 16 51 52](https://user-images.githubusercontent.com/4780/64034427-8ae62a80-cb46-11e9-95d8-63463cc62164.png)

**Plans**

Plans list and detail now support dark mode

![plans-dark](https://user-images.githubusercontent.com/4780/64034451-9a657380-cb46-11e9-8087-f8596baae9e3.png)

**To test:**

* Build and run on iOS 13 light and dark mode, and iOS 12
* Check the styles of each section listed above.

Update release notes:
